python/xref2vtysh.py

```diff
@@ -324,17 +324,7 @@ def process(cls, nodes, name, origin, spec):
     def load(cls, xref):
         nodes = NodeDict()
 
-        mgmtname = "mgmtd/libmgmt_be_nb.la"
```

Comment:
If the daemon is not NB converterd then it surely won't have it's function linked into libmgmt_be_nb.la right?

Reply:
There's a `match source-protocol` command in bgpd that fully matches the same command in zebra. And this code prevents `vtysh` from sending the command to bgpd.